src/NzbDrone.Common/Disk/FileAlreadyExistsException.cs

```diff
@@ -0,0 +1,14 @@
+using System;
+
+namespace NzbDrone.Common.Disk
+{
+    public class FileAlreadyExistsException : Exception
+    {
+        public string FilePath { get; set; }
```

Comment:
I think it makes sense to match the naming from `SameFilenameException`, `Filename` instead of `FilePath`